### ISSUE TITLE
Conda Yml file not present

### ISSUE THREAD

**Issue by PreyumKr** [CuiRuikai/Partial2Complete#34]
Conda create without any config is useless. There is no config file for the conda environment.

**Comment by CuiRuikai** [comment]
Apologies for the inconvenience, and thank you for bringing this to my attention. I’ll ensure this is corrected in future updates.

As a temporary solution, you can use pip to install the packages listed in the conda configuration file. It’s possible that I manually added the dependencies instead of using conda to export the environment, which may have led to this issue.

**Comment by PreyumKr** [comment]
ModuleNotFound error for Datasets module. Is it the huggingFace datasets module used inside the tools script or something else?

**Comment by CuiRuikai** [comment]
I did not use hugging face and I don’t think PyTorch used it. It might be an issues of indexing that your python did not find the dataset py file

**Comment by PreyumKr** [comment]
dataset.py named file is nowhere in the current git push of the repository

**Comment by CuiRuikai** [comment]
Yes, there is no such a dataset.py file. Could you provide the exact issues message

**Comment by PreyumKr** [comment]
okay that was error on my side. somehow the dataset folder was missing on my copy of the repo
